### DISCUSSION TITLE
How to compare archive with local filesystem, but don't compare file contents?

### DISCUSSION THREAD

**Discussion by mariomadproductions** [borgbackup/borg#8480] [Q&A]
The FAQ says you can use `borg export-tar` with tar --compare to compare an archive with the local filesystem, but from what I understand, this compares the file contents too. What's the correct way to a quicker check just based on file metadata (even just filepaths)? I just want to do a double check on the exclusions I chose for my backup command.

**Comment by ThomasWaldmann** [comment]
A good way to check excludes is to use `borg create --list --dry-run ...` while working on them or (for already in-production backups) the same without `--dry-run`.

**Reply by mariomadproductions** [reply]
Thanks. For existing backups, is there a better way than using `borg list` and then recreating the format in a `find` command, run that on the local filesystem, and compare the outputs?

**Reply by ThomasWaldmann** [reply]
You can use `--format` with `borg list` to create the format you want (limited by what borg offers as placeholder). Guess you might also need 2x `sort` command if you want to compare to find output.

**Reply by mariomadproductions** [reply]
Ah good point, I can also/instead change borg's format. I'll try that out.
